src/ui/localization/translator.ts

```diff
@@ -93,7 +94,7 @@ export class Translator {
         currency,
       }).format(price);
     } catch {
-      console.debug(
+      Logger.debugLog(
```

Comment:
Hmm I'm wondering if these should actually be `Logger.errorLog`... But we can keep as is for now.

Reply:
I can go ahead and make the change to `Logger.errorLog` if you give me a 👍🏻 . I initially did not want to change the behavior so did not even consider it.

Reply:
Let's do the change then 👍. I believe it's an error, so it makes more sense to me!